cmd/src/bin/tikv-ctl.rs

```diff
@@ -744,6 +748,10 @@ impl DebugExecutor for DebugClient {
             v1!("{}: {}", prop.get_name(), prop.get_value());
         }
     }
+
+    fn dump_store_info(&self) {}
+
+    fn dump_cluster_info(&self) {}
```

Comment:
Since this pr is only available in `Debugger`,  could you consider using 
```
    fn check_local_mode(&self) {
        ve1!("This command is only for local mode");
        process::exit(-1);
    }
```
or 
```
fn recover_all(&self, _: usize, _: bool) {
        unimplemented!("only available for local mode");
    }
```
Here?